package.json

```diff
@@ -15,6 +15,7 @@
     "death": "^1.0.0",
     "debug": "^2.2.0",
     "detect-indent": "^5.0.0",
+    "flow": "^0.2.3",
```

Comment:
I tried last friday to bump Flow's version to check if my issue was fixed in the new versions, but forgot to rollback the change.